lib/sshkit/ssh/connection.ex

```diff
@@ -11,23 +11,27 @@ defmodule SSHKit.SSH.Connection do
   """
 
   alias SSHKit.SSH.Connection
+  alias SSHKit.SSH.DryRun
   alias SSHKit.Utils
 
   defstruct [:host, :port, :options, :ref, :ssh_modules]
```

Comment:
Maybe just personal preference, but what do you think about including the Erlang modules as actual defaults in the struct definition?
```elixir
@ssh_modules %{ssh: :ssh, ssh_connection: :ssh_connection}
defstruct [:host, :port, :options, :ref, ssh_modules: @ssh_modules}
```